.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseCJKNumberParser.cs

```diff
@@ -111,6 +111,13 @@ public override ParseResult Parse(ExtractResult extResult)
                 }
             }
 
+            //// Added to determine the subtype
+            if (ret != null)
```

Comment:
Does it negatively affect Chinese? If so, please add a check for it.

Reply:
We'll double check and do so.